src/audio/rtnr/rtnr.c

```diff
@@ -240,7 +240,7 @@ static struct comp_dev *rtnr_new(const struct comp_driver *drv,
 
 	comp_set_drvdata(dev, cd);
```

Comment:
if possbile, please provide *why* this change is needed now and it worked fine before. this could offer some insights in the rtnr development.

otherwise, this looks good.

Reply:
Hi @dbaluta , thanks for the response. I've modified the commit and the description of this PR. Please review it again, thank you!

Reply:
@MingJenTai much better now. Please fix checkpatch error:

```
WARNING: Possible unwrapped commit description (prefer a maximum 75 chars per line)
```
and we are ready to go.

Reply:
Done : )

Reply:
all good now.